src/braket/default_simulator/openqasm/_helpers/quantum.py

```diff
@@ -75,6 +74,17 @@ def convert_phase_to_gate(controlled_phase: QuantumPhase) -> QuantumGate:
         controlled_phase.qubits,
     )
 
+    if first_ctrl_modifier.modifier == GateModifierName.negctrl:
+        X = QuantumGate(
+            [],
+            Identifier("U"),
+            [Identifier("π"), IntegerLiteral(0), Identifier("π")],
+            controlled_phase.qubits,
+        )
+        return [X, ctrl_phaseshift, X]
```

Comment:
I'm confused how this is being handled without any changes to [inline_gate_def_body](https://github.com/amazon-braket/amazon-braket-default-simulator-python/blob/main/src/braket/default_simulator/openqasm/interpreter.py#L321-L326)